pom.xml

```diff
@@ -27,8 +27,17 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
```

Comment:
Consider using <optional>true</optional> for the 'junit-jupiter-api' dependency. This will prevent the transitive dependency from being propagated to other modules that depend on this one.